rdmo/core/apps.py

```diff
@@ -0,0 +1,10 @@
+from django.apps import AppConfig
+from django.utils.translation import gettext_lazy as _
+
+
+class AccountsConfig(AppConfig):
+    name = 'rdmo.core'
+    verbose_name = _('Core')
+
+    def ready(self):
```

Comment:
The ready function does not need to be here, it should only be added when actually used.